astropy/modeling/functional_models.py

```diff
@@ -786,6 +786,53 @@ def inverse(self):
         return self.__class__(slope=new_slope, intercept=new_intercept)
 
 
+class Planar2D(Fittable2DModel):
```

Comment:
Good question. I dig back at the history at notice that `Linear1D` was added pretty early via #1203 as part of some other refactoring. So probably could have but did not happen.